### PR TITLE
Introduce a way to SIGSEGV perl

### PR DESCRIPTION
**Pull request by mkutlak**
Signed-off-by: Martin Kutlak <mkutlak@redhat.com>

### CODE REVIEW

File: src/will_perl_sigsegv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env perl
+kill 11,$$
```
Comment (ernestask) [inline src/will_perl_sigsegv]:
```suggestion
kill 'SEGV',$$
```

Could that be used instead?

Comment (ernestask) [reply src/will_perl_sigsegv]: https://perldoc.perl.org/functions/kill.html

**Comment by ernestask** [comment]
El gee tee em.

**Comment by mkutlak** [comment]
> El gee tee em.

Hah goteem

File: will-crash.spec.in [hunk 2]
```diff
@@ -18,10 +18,10 @@ BuildRequires:  java-devel
 BuildRequires:  libtool
 BuildRequires:  gcc-c++
 BuildRequires:  ruby-devel
-Requires:       ruby
 
+Requires:       ruby
+Requires:       perl
```
Comment (mkutlak) [inline will-crash.spec.in]: We don't need `perl`, but `perl-interpreter`.

**Pull request merged**